styled-components.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   extends: ["./index.js", "stylelint-config-styled-components"],
-  customSyntax: "@stylelint/postcss-css-in-js",
+  customSyntax: "postcss-styled-syntax",
```

Comment:
![memo](https://img.shields.io/badge/review-memo-white.svg)
[@stylelint/postcss-css-in-js](https://www.npmjs.com/package/@stylelint/postcss-css-in-js) is deprecated, so I use [postcss-styled-syntax](https://www.npmjs.com/package/postcss-styled-syntax) alternatively.

https://stylelint.io/migration-guide/to-15#removed-processors-configuration-property